data-store/diffusion-kafka-app-L2/README.md

```diff
@@ -37,8 +32,8 @@ We are going to map `kafka.firehose.fx` stream (we set up on previous step) to a
 
 ![](https://github.com/pushtechnology/tutorials/blob/master/data-store/diffusion-kafka-app-L2/images/topic%20views.png)
 
-## Step 4: Check the Topic View is multiplexing the Kafka firehose
```

Comment:
I think this is not very clear. The term “multiplexing” might be confusing to users.